src/java/org/apache/cassandra/utils/ObjectSizes.java

```diff
@@ -125,7 +124,7 @@ public static long sizeOnHeapOf(ByteBuffer buffer)
         // if we're only referencing a sub-portion of the ByteBuffer, don't count the array overhead (assume it's slab
         // allocated, so amortized over all the allocations the overhead is negligible and better to undercount than over)
```

Comment:
So this comment is still correct, but we've added the overhead of the `ByteBuffer` instance itself, which was missing?

Reply:
yep